src/types.ts

```diff
@@ -3,3 +3,14 @@ export interface RegisteredService {
   displayName: string;
   definition: any;
 }
+
+export type EventType = 'request' | 'response';
+export interface LoggerEvent {
```

Comment:
I have a concern about the creation of interface, that is specific for this component, in the place, where we use the component. I think that ideally a user should be able to export this interface from Logger component module.
@stephanebenayoun What do you think about it?